lib/settings_base.py

```diff
@@ -740,6 +740,7 @@ def JINJA_CONFIG():
             'js/zamboni/outgoing_links.js',
         ),
         'fxa': [
+            'js/lib/Uri.js',
```

Comment:
I don't think upper case filenames are a good idea because Mac OS X is case insensitive. I know we run everything through Docker but it seems risky nonetheless.